homeassistant/components/squeezebox/__init__.py

```diff
@@ -180,6 +199,21 @@ async def _discovered_player(player: Player) -> None:
     return True
 
 
+async def options_update_listener(
+    hass: HomeAssistant, entry: SqueezeboxConfigEntry
+) -> None:
+    """Handle options update."""
+
+    entry.runtime_data.options.browse_limit = entry.options.get(
+        CONF_BROWSE_LIMIT, DEFAULT_BROWSE_LIMIT
+    )
+    entry.runtime_data.options.volume_step = entry.options.get(
+        CONF_VOLUME_STEP, DEFAULT_VOLUME_STEP
+    )
+
+    await hass.config_entries.async_reload(entry.entry_id)
```

Comment:
Why do you update the runtime_data, if you are reloading anyway?

Reply:
It's a good question ;-)  Just left over from previously I guess.